OpenRA.Mods.Common/Scripting/Global/UserInterfaceGlobal.cs

```diff
@@ -31,5 +31,10 @@ public void SetMissionText(string text, Color? color = null)
 			var c = color.HasValue ? color.Value : Color.White;
 			luaLabel.GetColor = () => c;
 		}
+
+		public string GetTranslation(string text)
```

Comment:
Why `GetTranslation` when everything else uses `Translate`?

Reply:
Because of `GetPlayers`, `GetObjectiveDescription`, `GetActorsByType`, `GetGroundAttackers` but I can change it.